chainer/distributions/geometric.py

```diff
@@ -0,0 +1,73 @@
+import chainer
+from chainer.backends import cuda
+from chainer import distribution
+from chainer.functions.math import exponential
+
+
+class Geometric(distribution.Distribution):
+
+    """Geometric Distribution.
+
+    The probability mass function of the distribution is expressed as
+
+    .. math::
+        f(x = k) = p(1-p)^{k-1},
```

Comment:
- I'd write `f(k) =` (or `Pr(X = k) =` if we mention a random variable `X`).  Is the notation `f(x = k)` standard?
- Could you clarify that the support is "k = 1, 2, 3, ..." since there seems a school where the support is "k = 0, 1, 2, ...".